src/components/Auth-Components/input.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { h, FunctionComponent } from 'preact'
+
+interface InputProps {
+
```

Comment:
Are you using this file?

Reply:
It doesn't seem so, if not, delete it

Reply:
This component was created for the authentication pages that I had made initially. It has now gone unused since we are strictly using Github authentication with buttons. I can get rid of it.